### DISCUSSION TITLE
Can't run nicegui on sagemaker notebook instance

### DISCUSSION THREAD

**Discussion by yipeng-jiang** [zauberzeug/nicegui#1735] [General]
Hi, there. 
I am tring to run nicegui on the aws sagemaker notebook instance. I am  only able to get the inital rendering of the component, any interactions of the component is not reactive. From the console, I saw the error saying the following message.
`The client is using an unsupported version of the Socket.IO or Engine.IO protocols (further occurrences of this error will be logged with level INFO)`
After I changed the version to the same as my local computer which runs without any issue, it is still having the same problem.

**Comment by yipeng-jiang** [comment]
python-engineio==4.7.1
python-socketio==5.9.0

**Comment by falkoschindler** [comment]
Sorry, I don't know AWS SageMaker Notebook Instances and I'm struggling to understand which client version it is complaining about. The socket.io.min.js is served by NiceGUI itself and should match the server version 5.9.0.
Has anyone else experience with this platform?
